nilearn/datasets/func.py

```diff
@@ -1301,98 +1520,203 @@ def fetch_surf_nki_enhanced(n_subjects=10, data_dir=None,
-                                                     'U1', 'U1'],
-                               encoding=None)
+    phenotypic = np.genfromtxt(
+        phenotypic,
+        skip_header=True,
+        names=["Subject", "Age", "Dominant Hand", "Sex"],
+        delimiter=",",
+        dtype=["U9", "<f8", "U1", "U1"],
+        encoding=None,
+    )
 
     # Keep phenotypic information for selected subjects
     int_ids = np.asarray(ids)
-    phenotypic = phenotypic[[np.where(phenotypic['Subject'] == i)[0][0]
-                             for i in int_ids]]
+    phenotypic = phenotypic[
+        [np.where(phenotypic["Subject"] == i)[0][0] for i in int_ids]
+    ]
 
     # Download subjects' datasets
     func_right = []
     func_left = []
     for i in range(len(ids)):
-
-        archive = url + '%i/%s_%s_preprocessed_fsaverage5_fwhm6.gii'
-        func = os.path.join('%s', '%s_%s_preprocessed_fwhm6.gii')
-        rh = _fetch_files(data_dir,
-                          [(func % (ids[i], ids[i], 'right'),
-                           archive % (nitrc_ids[2*i+1], ids[i], 'rh'),
-                           {'move': func % (ids[i], ids[i], 'right')}
-                            )],
-                          resume=resume, verbose=verbose)
-        lh = _fetch_files(data_dir,
-                          [(func % (ids[i], ids[i], 'left'),
-                           archive % (nitrc_ids[2*i], ids[i], 'lh'),
-                           {'move': func % (ids[i], ids[i], 'left')}
-                            )],
-                          resume=resume, verbose=verbose)
+        archive = url + "%i/%s_%s_preprocessed_fsaverage5_fwhm6.gii"
+        func = os.path.join("%s", "%s_%s_preprocessed_fwhm6.gii")
+        rh = _fetch_files(
+            data_dir,
+            [
+                (
+                    func % (ids[i], ids[i], "right"),
+                    archive % (nitrc_ids[2 * i + 1], ids[i], "rh"),
+                    {"move": func % (ids[i], ids[i], "right")},
+                )
+            ],
+            resume=resume,
+            verbose=verbose,
+        )
+        lh = _fetch_files(
+            data_dir,
+            [
+                (
+                    func % (ids[i], ids[i], "left"),
+                    archive % (nitrc_ids[2 * i], ids[i], "lh"),
+                    {"move": func % (ids[i], ids[i], "left")},
+                )
+            ],
+            resume=resume,
+            verbose=verbose,
+        )
 
         func_right.append(rh[0])
         func_left.append(lh[0])
 
-    return Bunch(func_left=func_left, func_right=func_right,
-                 phenotypic=phenotypic,
-                 description=fdescr)
+    return Bunch(
+        func_left=func_left,
+        func_right=func_right,
+        phenotypic=phenotypic,
+        description=fdescr,
+    )
 
 
 @fill_doc
 def _fetch_development_fmri_participants(data_dir, url, verbose):
-    """Helper function to fetch_development_fmri.
+    """Fetch_development_fmri.
```

Comment:
```suggestion
    """Use in fetch_development_fmri function.
```

Or something like that